### PR TITLE
feat: replace client endpoint to sequencer

### PR DESCRIPTION
**Pull request by bonustrack**

### CODE REVIEW

File: src/constants.json [hunk 1]
```diff
@@ -1,3 +1,10 @@
 {
-  "version": "0.1.3"
+  "livenet": {
+    "hub": "https://hub.snapshot.org",
```
Comment (ChaituVR) [inline src/constants.json:3]: I get `snapshot.min.js:15          POST https://hub.snapshot.org/ 404` error when i try to vote @bonustrack

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/constants.json (hunk 1 above)
Comment (ChaituVR) [reply src/constants.json:3]: I use same code as here https://docs.snapshot.org/snapshot.js#init-client

**Comment by bonustrack** [comment]
Can you debug this PR @ChaituVR ?

**Review by ChaituVR — COMMENTED** [review]
There was a issue with `.replace` so it was not working. fixed it now 👍

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**